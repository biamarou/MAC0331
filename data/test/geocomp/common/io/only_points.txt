6 5
9 2
10 6
12 4
-7 1
-7 3
-9 2



# Commentary
# This should not be read




-9 -3
-7 -3
-8 -5




3 -4


8 -6



# One more commentary
