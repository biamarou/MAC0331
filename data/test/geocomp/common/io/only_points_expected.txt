6 5
9 2
10 6
12 4
-7 1
-7 3
-9 2
-9 -3
-7 -3
-8 -5
3 -4
8 -6
